torchtext/prototype/models/t5/model.py

```diff
@@ -136,12 +131,24 @@ def __init__(
             for p in self.parameters():
                 p.requires_grad = False
 
+    def prepare_inputs_for_generation(self, input_ids, encoder_outputs, **kwargs):
```

Comment:
@Nayef211 Any advice on what to do here? I know `**kwargs` are disallowed by TorchScript but I don't know of an elegant way to generalize the generation utils without allowing kwargs.

Reply:
+1